pyproject.toml

```diff
@@ -151,8 +151,6 @@ confluent-kafka = ">=1.9.2,<3.0.0"
 portalocker = "^2.7.0"
 structlog = "^23.1.0"
 structlog-sentry = "^2.0.2"
-# pin dnspython to avoid dependency incompatibility
-# in order to fix https://rasahq.atlassian.net/browse/ATO-1419
```

Comment:
Can we keep these comments? it's quite a pain that poetry removes them automatically when it runs commands that updates pyproject 🙏🏻